README.md

```diff
@@ -19,6 +19,120 @@ let package = Package(
 )
 ```
 
+## Example
+In this example we'll build a simple "reverse-echo" client and server that utilizes a secure SSL socket for sending and receiving messages.
+
+### Client
+The client connects to a server available at `serverFQDN:8443` and sends a string.  It then waits for a response from the server and then exits.
```

Comment:
Perhaps its best to hardcode a server that everybody can connect to? Something like apple.com.